.github/workflows/stale.yml

```diff
@@ -0,0 +1,21 @@
+name: "Mark/Close Stale Issues/PRs"
+on:
+  schedule:
+  - cron: "* */1 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v1.1.0
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: |-
+          This issue is stale because it has been open 180 days with no activity. Remove stale label or comment on the issue or it will be closed in 30 days.
+        stale-pr-message: |-
+          This pull request is stale because it has been open 180 days with no activity. Remove stale label or comment on the pull request or it will be closed in 30 days.
```

Comment:
Do we want to change these messages?

Reply:
I suggest just a minor rewording.  I would rather they say something rather than simply remove the label:
```
This issue has been labeled stale because it has been open for 180 days with no activity.  It will be closed automatically in 30 days unless you either comment in the issue (preferred) or remove the stale label.
```
```

Reply:
Actually, can a non-committer remove a label?  If not I think that text should be deleted.

Reply:
A non-committer cannot remove labels. That is only committers can remove labels. Contributors (anyone that can comment on GitHub) only have the option of commenting to prevent automatic deletion.

>  If not I think that text should be deleted.

Which text specifically should we delete?